src/DotVVM.Framework/Utils/ReflectionUtils.cs

```diff
@@ -63,6 +64,12 @@ public static IEnumerable<MemberInfo> GetAllMembers(this Type type, BindingFlags
                 return type.GetMembers(flags);
         }
 
+        public static IEnumerable<MemberInfo> GetAllExtensions(this Type type, BindingFlags flags = BindingFlags.Public | BindingFlags.Static)
+        {
+            foreach (var registeredType in TypeRegistry.GetRegisteredTypesForExtensionMethodsLookup())
+                foreach (var member in registeredType.GetMembers(flags))
+                    yield return member;
```

Comment:
Does this return something else than method?

Reply:
No - it should return only methods. Changed to `MethodInfo`